packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -246,6 +246,64 @@ describe('Validation at REST level', () => {
     });
   });
 
+  context(
+    'with request body validation options - {ajvTranslate: [Function]}',
+    () => {
+      class ProductController {
+        @post('/products')
+        async create(
+          @requestBody({required: true}) data: Product,
+        ): Promise<Product> {
+          return new Product(data);
+        }
+      }
+
+      before(() =>
+        givenAnAppAndAClient(ProductController, {
+          nullable: false,
+          compiledSchemaCache: new WeakMap(),
+          $data: true,
+          ajvTranslate: errors => {
+            return errors.map(e => ({
+              ...e,
+              message: `(translated) ${e.message}`,
+            }));
+          },
+        }),
+      );
+      after(() => app.stop());
+
+      it('rejects requests with translated error message', async () => {
```

Comment:
I read this test name as "the framework reject requests when the client provided a translated error message". Let's fine a better one please. For examples:

```suggestion
      it('transforms error messages provided by AJV', async () => {
```